dev/test.ts

```diff
@@ -105,7 +106,7 @@ const env: Record<string, string> = {
   "TMP_DIR": tmpDir,
   "TIMER_MAX_TIMEOUT_MS": "30000",
   "NPM_CONFIG_REGISTRY": "http://localhost:4873",
-  "PATH": `${Deno.env.get("PATH")}:${join(projectDir, "target/debug")}`,
+  "PATH": `${join(projectDir, "target/debug")}:${Deno.env.get("PATH")}`,
```

Comment:
This line took me 3 hours lol.

Reply:
jz... might add a comment for the next change there?